Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json

```diff
@@ -7966,12 +7928,7 @@
 													"});\r",
 													"\r",
 													"pm.test(\"Should return a message indicating that FiscalYear is required.\", () => {\r",
-													"  pm.expect(problemDetails.validationErrors).to.deep.equal({\r",
-													"    \"$.fiscalYear\": [\r",
-													"      \"FiscalYear must be between 2020 and 2040.\",\r",
-													"      \"Value is required.\"\r",
-													"    ]\r",
-													"  });\r",
+													"  pm.expect(problemDetails.validationErrors[\"$.fiscalYear\"]).to.match(/(Value|FiscalYear) is required./i);\r",
```

Comment:
The current API implementation uses a general purpose message of "Value is required", however using the element name instead of "Value" (e.g. FiscalYear is required.) is also acceptable.